### PR TITLE
Adding close loop of bi_purge test (BZ 1910503) to rhcs-5.3 specific fixes.

### PR DESCRIPTION
**Pull request by viduship**
This PR is for RHCS-5.3 specific fixes and also addresses the tests to be executed in parallel.

Additionally it also addresses a new close-loop: Adding test for bi purge bug-1910503

Logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-NQ3GVK

Signed-off-by: viduship <vimishra@redhat.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: pipeline/metadata/5.3.yaml [hunk 1]
```diff
@@ -1028,8 +1028,8 @@
     - rgw
     - stage-1
 
-- name: "test-empty-bucket-notification"
-  suite: "suites/pacific/rgw/tier-2_rgw_test-empty-bucket-notifcations.yaml"
+- name: "test-5.3-specific-fixes"
+  suite: "suites/pacific/rgw/tier-2_rgw_test_5.3_specific_fixes.yaml"
```
Comment (udaysk23) [inline pipeline/metadata/5.3.yaml:1032]: Are we not running `tier-2_rgw_test-empty-bucket-notifcations.yaml` suite on RHCS 5.3?

**Review by udaysk23 — DISMISSED** [review]

File: pipeline/metadata/5.3.yaml (hunk 1 above)
Comment (anrao19) [inline pipeline/metadata/5.3.yaml:1032]: Is there any specific reason that, suite: tier-2_rgw_test-empty-bucket-notifcations.yaml removed from pipeline execution?

Comment (viduship) [reply pipeline/metadata/5.3.yaml:1032]: It is not removed, just the file has been renamed @anrao19

Comment (viduship) [reply pipeline/metadata/5.3.yaml:1032]: the new file tier-2_rgw_test_5.3_specific_fixes.yaml is having 2 tests that are only fixed in rhcs-5.3 "test bucket notifcation with empty configuration" and  "test bi purge"

Comment (anrao19) [reply pipeline/metadata/5.3.yaml:1032]: @viduship, 
suite : tier-2_rgw_test_5.3_specific_fixes.yaml is a new file not a renamed suite of tier-2_rgw_test-empty-bucket-notifcations.yaml right? 
then Changes in file pipeline/metadata/5.3.yaml, looks like the existing suite tier-2_rgw_test-empty-bucket-notifcations.yaml removed from pipeline execution.

Comment (viduship) [reply pipeline/metadata/5.3.yaml:1032]: Suite will be removed, but the test case is there in the newly named suite file

Comment (anrao19) [reply pipeline/metadata/5.3.yaml:1032]: IMO, If suite removed or renamed in same pr, that would have been more understandable

Comment (viduship) [reply pipeline/metadata/5.3.yaml:1032]: Yeah, I agree. Consider it as part of the current activity of identifying suites for parallel execution.

Also, since this suite is not just specific to testing empty bucket notifications, hence the rationale is to rename it as it also has other tests.

Comment (viduship) [reply pipeline/metadata/5.3.yaml:1032]: Shall we proceed to resolve it @anrao19 ?

Comment (anrao19) [reply pipeline/metadata/5.3.yaml:1032]: sure @viduship

**Review by anrao19 — APPROVED** [review]

File: pipeline/metadata/5.3.yaml (hunk 1 above)
Comment (viduship) [reply pipeline/metadata/5.3.yaml:1032]: @anrao19 , I have removed the old suite file before renaming it.

**Review by mkasturi18 — APPROVED** [review]
